app/views/clearing-house/record-list.vue

```diff
@@ -329,6 +335,14 @@ export default {
         }
     },
     methods : {
+        async triggerReindex(schema, identifier){
+            const params = {
+                'schema': schema, 
+                'identifier': identifier
+            }
+            const solrResponse = await solrIndexAPI.reIndex(params)
+            console.log("Re-index response :", solrResponse)
```

Comment:
1. use the toast to show error or success message. 
2. hide the reindex button after success 
3. show refresh button to get updated info from index